modules/managed-cluster-add-notification-contacts.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-notifications.adoc
+// * osd_cluster_admin/osd-cluster-notifications.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="add-notification-contact_{context}"]
+= Adding notification contacts to your cluster
+
+Notification contacts receive emails when cluster notifications are sent to the cluster.
+By default, only the cluster owner receives cluster notification emails. You can configure additional notification contacts in your cluster support settings.
+
+.Prerequisites
+* Your cluster is deployed and registered to the {hybrid-console}.
+* You are logged in to the {hybrid-console-second}.
+
+.Procedure
+. Navigate to the Clusters page of the {hybrid-console-second}.
+. Click the name of your cluster to go to the cluster details page.
+. Click the **Support** tab.
+. On the **Support** tab, find the **Notification contacts** section.
+. In the **Address** field, enter the email address of the new recipient.
+//. In the type field, select the types of cluster notification to send to this recipient.
```

Comment:
@laubai Steps seems wrong. Under "Notification contacts" , user need to click a button "Add notification contact". A new dialog will appear with a field "RedHat username or email" as shown below.
![image](https://github.com/openshift/openshift-docs/assets/93653094/b9631dac-1a3a-44d1-bad3-743465c3b03e)
User could specify the email or username then click "Add contact" to add the notification contacts. Please review and update this section.

Reply:
Thanks, I've updated now!